Curvature/HTTPRequest/HTTPRequest.swift

```diff
@@ -32,6 +32,7 @@ public struct HTTPRequest {
 
     public var parameters: [String: String] = [:]
     public var data: [String: Any] = [:]
+    public var context: [String: Any] = [:]
```

Comment:
Hmm, that's a good point. I don't think XMLs map very well to dictionaries, but URL encoded bodies do, so this is indeed very interesting.